WSL instructions

```diff
@@ -0,0 +1,125 @@
+sudo -i
+apt -y install tmux
+
+sudo add-apt-repository "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) main universe multiverse restricted"
+apt -y install curl
+curl -sL https://deb.nodesource.com/setup_8.x | bash -
+curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update && sudo apt-get install yarn
+
+apt -y install imagemagick ffmpeg libpq-dev libxml2-dev libxslt1-dev file git-core g++ libprotobuf-dev protobuf-compiler pkg-config nodejs gcc autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm5 libgdbm-dev nginx redis-server redis-tools postgresql postgresql-contrib certbot yarn libidn11-dev libicu-dev
+
+
+adduser mastodon
+sudo su - mastodon
+
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+cd ~/.rbenv && src/configure && make -C src
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+exec bash
+type rbenv
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+
+RUBY_CONFIGURE_OPTS=--with-jemalloc rbenv install 2.6.1
+rbenv global 2.6.1
+gem update --system
+gem install bundler --no-document
+
+
+cd ~
+git clone https://github.com/tootsuite/mastodon.git live
+cd ~/live
+git checkout $(git tag -l | grep -v 'rc[0-9]*$' | sort -V | tail -n 1)
+gem install bundler
+bundle install -j$(getconf _NPROCESSORS_ONLN) --deployment 
+gem install rubocop-performance
+gem install rubocop-rails
+yarn install --pure-lockfile
```

Comment:
yarn install --skip-integrity-check # use this if error6 ENOENT: no such file or directory, lstat